File: Convert different pi numbers to new pi/Old_pi_to_new_pi.java
import java.util.Scanner;

public class Old_pi_to_new_pi{
    public static void main(final String[] args) {
        // Number used as diamiter
        final Double diamiter = 10.0;
        // Radius number
        final Double radius = diamiter/2;
        // Scanner Decleration
        final Scanner input = new Scanner(System.in);

        // First displayed line
        System.out.println("Input the variarion of pi you want to convert to new pi");
        // Input under the line
        final Double pi_num_var = input.nextDouble();
        // Calculation
        final Double calculation1 = ((radius * radius) * pi_num_var) / 100;
        // Display Result
        System.out.println("New pi is: " + calculation1);
        System.out.println("Thank you for using Seiling Old to new pi converter");
    } 
}

File: Convert different pi numbers to new pi/Old_pi_to_new_pi.cpp
#include <iostream>

using namespace std;

int main(){
    // Number used as diameter
    double diamiter = 10;

    // Radius variable
    double radius = diamiter / 2;

    // Stating of input request
    cout << "input variation of pi you wand to preform the calculation on:" << endl;

    // Input var deceleration
    double pi;

    // Input usage
    cin >> pi;

    // Calculation
    double cal1  = ((radius * radius)* pi) / 100;

    // Display result
    cout << "The new pi is: " + to_string(cal1) << endl;
    cout << "Thank you for using Seiling old pi to new pi converter." << endl;
    return 0;
}